docs/user-guide/index.md

```diff
@@ -35,4 +36,5 @@ There are a number of classes of HELICS users:
 *   [Installation](../installation/index.html) -  Instructions on how to install HELICS
 *   [C API](../c-api-reference/index.html)
 *   [Developer's Guide](../developer-guide/index.html) - Details on how the software is assembled and some of the underlying components.
+*   [Existing Tools](../Tools_using_Helics.md) - List of the existing tools using HELICS and some under development. 
```

Comment:
```
*   [Existing Tools](../Tools_using_HELICS.md) - List of the existing tools using HELICS and some under development. 
```

Reply:
pretty sure they are not on windows and are elsewhere.  This has been corrected.